integrations/PinterestTag/browser.js

```diff
@@ -1,31 +1,28 @@
 /* eslint-disable class-methods-use-this */
-import get from "get-value";
-import logger from "../../utils/logUtil";
+import get from 'get-value';
```

Comment:
By default we use `""` not `'` can we please update it?

Reply:
We are now slowly moving away from using `"`. It's been active on v1.1 branches now.

Reply:
Ok